modules/c++/re/source/RegexPCRE.cpp

```diff
@@ -20,39 +20,41 @@
  *
  */
 
-#if defined(USE_PCRE)
+#include "re/Regex.h" // this has to come before the #ifdef checks below
```

Comment:
Oops, nevermind. That was actually because the __CODA_CPP11 define is #included from sys_config.h in Regex.h, so that has to be #included before the #ifdef.